### ISSUE TITLE
Cannot pickle "configModuleInstance" object

### ISSUE THREAD

**Issue by Pwhsky** [DeepTrackAI/deeplay#134]
Hi, 

I am having an issue when instancing multiple deeplay models in the same notebook,
specifically  GraphToEdgeMAGIK followed by a LodeSTAR model. 

The MAGIK model is instanced without any errors, as well as the binary classifier used for training: 
(I have also tried using build() for the classifier with no success)

```
model = dl.GraphToEdgeMAGIK([96,] * 4,1, out_activation=torch.nn.Sigmoid)
model.encoder[0].configure(hidden_features=[32, 64],
                           out_features=96, 
                           out_activation=torch.nn.ReLU
)

model.encoder[1].configure(hidden_features=[32, 64],
                           out_features=96, 
                           out_activation=torch.nn.ReLU
)
model.head.configure(hidden_features=[64, 32])

classifier = dl.BinaryClassifier(model=model,
                                 optimizer=dl.Adam(lr=1e-3)
)
classifier = classifier.create()
```

Training then takes place with:
```
trainer = dl.Trainer(max_epochs=100)
trainer.fit(classifier,train_loader)
```
Then, instancing the LodeSTAR model at a later point in the notebook with:
```

lodestar_exp = dl.LodeSTAR(n_transforms=8,
                   optimizer=dl.Adam(lr=1e-3)).build()

```

Throws  "TypeError: cannot pickle 'ConfigModuleInstance' object", I append the cell output below:
![figure_1](https://github.com/user-attachments/assets/2b318cf4-aab5-4742-be3b-6562197d3f61)

Let me know if I should provide more context, thank you.

**Comment by BenjaminMidtvedt** [comment]
Hi @Pwhsky ! We are aware of this issue. It is due to an update to pytorch. The fix has already been implemented in PR #130 and will be pushed to pip today

**Comment by Pwhsky** [comment]
That is great news, thank you for the quick reply!
